platform_design/lib/utils.dart

```diff
@@ -26,26 +22,9 @@ const _myListOfRandomColors = [
 
 final _random = Random();
 
-final wordPairIterator = generateWordPair();
-Iterable<WordPair> generateWordPair() sync* {
-  bool filterWord(String word) => unsafe.contains(word);
-  String pickRandom(List<String> list) => list[_random.nextInt(list.length)];
-
-  String prefix;
-  while (true) {
-    if (_random.nextBool()) {
-      prefix = pickRandom(adjectives);
-    } else {
-      prefix = pickRandom(nouns);
-    }
-    final suffix = pickRandom(nouns);
-
-    if (filterWord(prefix) || filterWord(suffix)) continue;
-
-    final wordPair = WordPair(prefix, suffix);
-    yield wordPair;
-  }
-}
+// Avoid customizing the word generator, which can be slow.
+// https://github.com/filiph/english_words/issues/9
+final wordPairIterator = generateWordPairs();
```

Comment:
I think I did the above to be able to have non-default syllable lengths since the default lengths were too short aesthetically.

Reply:
Ah, I wasn't paying attention to that. My thinking was that it would be simpler to use the default behavior